docs/overview/queries/elastic_query_exists.md

```diff
@@ -3,4 +3,22 @@ id: elastic_query_exists
 title: "Exists Query"
 ---
 
-TBD
+The `Exists` query is used for returning documents that contain an indexed value for a field.
+
+In order to use the `Exists` query import the following:
+```scala
+import zio.elasticsearch.query.ExistsQuery
+import zio.elasticsearch.ElasticQuery._
+```
+
+You can create an `Exists` query using the `exists` method this way:
+```scala
+val query: ExistsQuery = exists(field = "name")
+```
+
+Also, you can create a type-safe `Exists` query using the `exists` method this way:
```

Comment:
Maybe for these type-safe methods there should be also link that leads you to page that shows how can Schema be created and used to create accessors. What do you think?